app/services/doc_auth/error_generator.rb

```diff
@@ -185,6 +189,11 @@ def get_error_messages(response_info)
         end
       end
 
+      portrait_match_results = response_info[:portrait_match_results] || {}
+      if !!liveness_enabled && portrait_match_results&.dig(:FaceMatchResult) != 'Pass'
```

Comment:
do we need '!!' here ... seems easier to read without it 🤔

Reply:
@amirbey , removed, it was to make sure it's a boolean.

Reply:
Maybe we should make those guarantees where we first assign it, so we can be sure that we're passing around a boolean and not something different?

i.e. 

```diff
diff --git a/app/services/doc_auth/error_generator.rb b/app/services/doc_auth/error_generator.rb
index 6060b4e1e..c55a3d636 100644
--- a/app/services/doc_auth/error_generator.rb
+++ b/app/services/doc_auth/error_generator.rb
@@ -64,7 +64,7 @@ module DocAuth
       image_metric_errors = get_image_metric_errors(response_info[:image_metrics])
       return image_metric_errors.to_h unless image_metric_errors.empty?
 
-      liveness_enabled = response_info[:liveness_enabled]
+      liveness_enabled = !!response_info[:liveness_enabled]
       alert_errors = get_error_messages(liveness_enabled, response_info)
       alert_error_count += 1 if alert_errors.include?(SELFIE)
 ```